packages/office-ui-fabric-react/src/components/pickers/BasePicker.styles.ts

```diff
@@ -9,38 +10,59 @@ const GlobalClassNames = {
 };
 
 export function getStyles(props: IBasePickerStyleProps): IBasePickerStyles {
-  const { className, theme, isFocused, inputClassName } = props;
+  const { className, theme, isFocused, inputClassName, disabled } = props;
 
   if (!theme) {
     throw new Error('theme is undefined or null in base BasePicker getStyles function.');
   }
-  const { semanticColors } = theme;
+  const { semanticColors, palette } = theme;
   const { inputBorder, inputBorderHovered, inputFocusBorderAlt } = semanticColors;
 
   const classNames = getGlobalClassNames(GlobalClassNames, theme);
 
+  // The following lines are to create a semi-transparent color overlay for the disabled state with designer's approval.
+  const rgbColor: IRGB | undefined = cssColor(palette.neutralQuaternaryAlt);
+  const disabledOverlayColor = rgbColor ? `rgba(${rgbColor.r}, ${rgbColor.g}, ${rgbColor.b}, 0.29)` : 'transparent';
```

Comment:
this is a bit confusing. Could we do this in a more static way?

Reply:
It is an expensive calculation; it'd be better to use a non calculated color.

Reply:
@dzearing I used the palette color in case it was themed. Not sure how would this look in a dark theme in case we use a static color. Let me talk to design again and see what can be done in here instead.

Reply:
Made the change and left a `todo` comment in case we are having issues with theming the color in dark themes.